README.md

```diff
@@ -1,4 +1,240 @@
+case a summary from the meeting will be posted back to the RFC pull request.
+
+A working group makes final decisions about RFCs after the benefits and drawbacks
+are well understood. These decisions can be made at any time, but the working group
+will regularly issue decisions. When a decision is made, the RFC pull request
+will either be merged or closed. In either case, if the reasoning is not clear
+from the discussion in thread, the working group will add a comment describing the
+rationale for the decision.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right
+away. Other accepted RFCs can represent features that can wait until some
+arbitrary developer feels like doing the work. Every accepted RFC has an
+associated issue tracking its implementation in the Rust repository; thus that
+associated issue can be assigned a priority via the triage process that the
+teams use for all issues in the IPFS repositories.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC
+author (like any other developer) is welcome to post an implementation for
+review after the RFC has been accepted.
+
+If you are interested in working on the implementation for an "active" RFC, but
+cannot determine if someone else is already working on it, feel free to ask
+(e.g. by leaving a comment on the associated issue).
+
+
+## RFC Postponement
+[RFC Postponement]: #rfc-postponement
+
+Some RFC pull requests are tagged with the "postponed" label when they are
+closed (as part of the rejection process). An RFC closed with "postponed" is
+marked as such because we want neither to think about evaluating the proposal
+nor about implementing the described feature until some time in the future, and
+we believe that we can afford to wait until then to do so. Historically,
+"postponed" was used to postpone features until after 1.0. Postponed pull
+requests may be re-opened when the time is right. We don't have any formal
+process for that, you should ask members of the relevant working group.
+
+Usually an RFC pull request marked as "postponed" has already passed an
+informal first round of evaluation, namely the round of "do we think we would
+ever possibly consider making this change, as outlined in the RFC pull request,
+or some semi-obvious variation of it." (When the answer to the latter question
+is "no", then the appropriate response is to close the RFC, not postpone it.)
+
+
+### Help this is all too informal!
+[Help this is all too informal!]: #help-this-is-all-too-informal
+
+The process is intended to be as lightweight as reasonable for the present
+circumstances. As usual, we are trying to let the process be driven by
+consensus and community norms, not impose more structure than necessary.
+
+
+[ipfs discussion forum]: https://discuss.ipfs.io/
+[RFC issue tracker]: https://github.com/ipfs/rfcs/issues
```

Comment:
This repo is called `RFC` and not `rfcs`. Though I'd prefer the lower case plural version.

Reply:
Agree. The repository has bunch of RFCs in it. The repository is not a RFC.

Reply:
I think someone changed the name of the repo on me. Not sure who did it or how it happened. @lgierth did you change it to RFC?

Reply:
✅ renamed the repo